recipes/cpr/all/conanfile.py

```diff
@@ -40,13 +44,23 @@ def _build_subfolder(self):
     def _supports_openssl(self):
         # https://github.com/whoshuu/cpr/commit/b036a3279ba62720d1e43362d32202bf412ea152
         # https://github.com/whoshuu/cpr/releases/tag/1.5.0
-        return tools.Version(self.version) >= "1.5.0"
+        return tools.Version(self.version) >= "1.5.0" and not tools.is_apple_os(self.settings.os)
 
     @property
     def _supports_winssl(self):
         # https://github.com/whoshuu/cpr/commit/18e1fc5c3fc0ffc07695f1d78897fb69e7474ea9
         # https://github.com/whoshuu/cpr/releases/tag/1.5.1
-        return tools.Version(self.version) >= "1.5.1"
+        return tools.Version(self.version) >= "1.5.1" and self.settings.os == "Windows"
+
+    @property
+    def _supports_darwinssl(self):
+        # https://github.com/whoshuu/cpr/releases/tag/1.6.1
+        return tools.Version(self.version) >= "1.6.1"
```

Comment:
```suggestion
        return tools.Version(self.version) >= "1.6.1" and tools.is_apple_os(self.settings.os)
```

Reply:
I read this change and I could have _SWORN_ I already did it. Guess not, lol. On it.